### PR TITLE
Fixes post update 2019-09-06

### PR DESCRIPTION
**Pull request by 3vcloud**
Adjusted player_email offset for CharContext
district_number to be found in WorldContext instead of CharContext

### CODE REVIEW

File: Source/MapMgr.cpp [hunk 1]
```diff
@@ -158,7 +158,7 @@ namespace GW {
     }
 
     int Map::GetDistrict() {
-        CharContext *ctx = GameContext::instance()->character;
+        WorldContext* ctx = GW::GameContext::instance()->world;
```
Comment (ldufr) [inline Source/MapMgr.cpp]: Why this change? Is `CharContext::district_number` not good anymore ?

File: Include/GWCA/Context/WorldContext.h [hunk 2]
```diff
@@ -76,7 +76,7 @@ namespace GW {
         /* +h062C */ Array<void *> h062C;
         /* +h063C */ uint8_t h063C[68];
         /* +h0680 */ Array<void *> h0680; // Struct size = 0x138 i think, has a vtable
-        /* +h0690 */ uint32_t salvage_session_id;
+        /* +h0690 */ uint32_t district_number;
```
Comment (ldufr) [inline Include/GWCA/Context/WorldContext.h]: Hum, that's odd. Are you 100% on that change?

Comment (3vcloud) [reply Include/GWCA/Context/WorldContext.h]: Yeh I intercepted the `InstanceLoadInfo` StoC packet and overrode the district number with something like 112, then checked which variable it was updating.

File: Source/MapMgr.cpp (hunk 1 above)
Comment (3vcloud) [reply Source/MapMgr.cpp]: I noticed that Trade alerts in toolbox were triggering even when in Kama ae1 - this was because `CharContext::district_number` is always 0 now ( I think the devs changed this in relation to the update notes from the other day - https://wiki.guildwars.com/wiki/Game_updates )

File: Include/GWCA/Context/WorldContext.h (hunk 2 above)
Comment (3vcloud) [reply Include/GWCA/Context/WorldContext.h]: Ah no this isn't right, will take another look and update with another commit

**Comment by 3vcloud** [comment]
Ok sorry about that, put the offset change in the wrong place. I've rolled back the changes to `MapMgr.cpp` and `WorldContext.h`, and I also found that `h224` offset is actually the map language.
